File: JavaScript/Resursion/Factorial.js
const prompt = require('prompt-sync')({sigint:true});

const n = parseInt(prompt("Enter N: "));

const Fact = (N) =>{
    if(N === 1 || N ==0 ){
        return 1;
    }
    return Fact(N-1) * N ;
}
const Res = Fact(n);
console.log(n+"! = "+Res);

File: C++ Language/Resursion/Factorial.cpp
#include<iostream>
using namespace std;

long Fact(long N){
    if (N == 0|| N== 1)
    {
        return 1;
    }
    return Fact(N-1) * N;
    
}
int main(void){
    int n;
    cout << "Enter N :" ;
    cin >> n;
    long res = Fact(n);
    cout << n << "! = " << res;  
}